src/components/chats/index.js

```diff
@@ -0,0 +1,71 @@
+import useTranslation from 'hooks/useTranslation';
+import { useEffect, useState } from 'react';
+import { useGetConversationQuery } from 'services/conversation/conversation';
+import profile from 'assets/profile.svg';
+import { v4 as uuid } from 'uuid';
```

Comment:
It's a good practice to always import React first, and you could change the order of imports to something like this:
1 - React
2 - External libraries (axios, uuid, Bootstrap etc)
3 - Relative imports, components, hooks, utils etc.
4 - Assets (images, videos, css etc)

For example
```suggestion
import { useEffect, useState } from 'react';

import { v4 as uuid } from 'uuid';

import useTranslation from 'hooks/useTranslation';
import { useGetConversationQuery } from 'services/conversation/conversation';

import profile from 'assets/profile.svg';
```